es-ES.js

```diff
@@ -1,150 +1,150 @@
 		noPiracy:
-			'No piracy will be allowed. No use of copyrighted assets is accepted, games must be the author original creation. We do allow the use of generic free assets you can find in unity assets store for example but do note that we will value more games which use hand made assets. As for ports, do not include game files if they are not free to share. Engine made from RE are allowed, but engine made from leaked source of a game engine are not allowed.',
-		paypal: 'You must have a paypal account to receive the money.',
+			'No se permite la piratería. No se acepta el uso de bienes con copyright, los juegos deben ser creaciones originales. Permitimos el uso de bienes genéricos gratuitos, por ejemplo, los puedes encontrar en la tienda de Unity per se valorará más los bienes hechos a mano. Para los ports, no incluyas los datos del juego si no se pueden compartir. Motores hechos con RE están permitidos, pero motores hechos con con código fuente filtrado de un motor de juego no está permitido.',
+		paypal: 'Debes de tener una cuenta de paypal para recibir el premio.',
 		splashScreen:
-			'You must include the contest splash screen in the booting process of your homebrew (you can include it as a live area asset Plugins obviously can’t show the splashcreen, instead we will ask you to put a mention of the kyûHEN contest inside your readme in your repo.',
+			'Tienes que incluir la pantalla del concurso en el proceso de arranque de tu homebrew (puedes incluirlo en el asset del live area. Los plugins obviamente no pueden mostrar la pantalla, así que te pedimo que menciones el concurso kyûHEN dentro del léeme de tu repositorio.',
 		publicRelease:
-			'You can share a public build outside of our website only when we have approved it and already showcased your project on our website',
+			'Puedes publicar versiones públicas fuera de nuestra web cuando hayamos comprobado que ya has mostrado tu proyecto en nuestra web',
 		openSource:
-			'If your project is based on an open source project, you must follow the license of the open source project',
-		cheats: 'If we catch you cheating, you are automatically banned from the contest.',
+			'Si tu proyecto está basado en un proyecto de código abierto, debes seguir usando la licencia del proyecto de código abierto',
+		cheats: 'Si comprobamos que haces trampas, serás expulsado del concurso de inmediato.',
 		categoriesPrizes:
-			'Categories prizes might be subject to slight changes depending on submissions and on the amount collected.',
+			'Los premios de las categorías pueden estar sujetos a pequeños cambios dependiendo de los proyectos y de la cantidad monetaria recolectada.',
 	},
 
 	pages: {
 		home: {
-			title: 'Home',
-			slogan: 'Who can brew the best',
+			title: 'Inicio',
+			slogan: 'El que pueda crear lo mejor',
```

Comment:
'Quién puede crear mejor?' would be the correct translation
Still, don't fucking translate an slogan